plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -331,11 +341,36 @@ func (e *quotaEvaluator) checkRequest(quotas []api.ResourceQuota, a admission.At
 		return quotas, nil
 	}
 
+	// if we have limited resources enabled for this resource, always calculate usage
+	inputObject := a.GetObject()
+	mustHaveCoveringQuota := []api.ResourceName{}
+	for _, limitedResource := range e.config.LimitedResources {
+		limitedGroupResource := schema.GroupResource{Group: limitedResource.APIGroup, Resource: limitedResource.APIResource}
+		if limitedGroupResource == a.GetResource().GroupResource() {
+			deltaUsage, err := evaluator.Usage(inputObject)
+			if err != nil {
+				return quotas, nil
+			}
+			for k, v := range deltaUsage {
+				// if a resource is consumed, we need to check if it matches on the limited resource list.
+				if v.Sign() == 1 {
+					match, err := regexp.MatchString(limitedResource.MatchExpression, string(k))
```

Comment:
Ok, now I see how the field is used.  You need an example in the godoc for the type.